config/grafana/dashboards/integration-service-dashboard.json

```diff
@@ -1271,6 +1271,100 @@
       "timeShift": null
     }
 
+    {
+      "id": 28,
+      "gridPos": {
+        "h": 8,
+        "w": 12,
+        "x": 12,
+        "y": 42
+      },
+      "type": "graph",
```

Comment:
Why not use heatmap for express the Histogram type metric? It seems more clear.

Reply:
done. thank you.